src/status_im/ui/screens/discover/subs.cljs

```diff
@@ -18,10 +19,10 @@
 
 (defn- get-discoveries-by-tags [discoveries current-tag tags]
   (let [tags' (or tags [current-tag])]
-    (filter #(every? (->> (:tags %)
-                          (map :name)
-                          (into (hash-set)))
-                     tags')
+    (filter #(some (->> (:tags %)
+                        (map :name)
+                        (into (hash-set)))
```

Comment:
Nitpicking, by I think is more idiomatic to just use reader literal `#{}` instead of the constructor function `(hash-set)`

Reply:
I agree, but we'll likely replace a lot of that code in the coming changes, so no point micro refactoring it now.